cms/static/sass/xmodule/_headings.scss

```diff
@@ -32,9 +32,8 @@ $headings-base-color:                       $gray-d2;
 
 
 %hd-2 {
-    margin-bottom: 1em;
-    font-size: 1.5em;
-    font-weight: $headings-font-weight-normal;
+    font-size: em(18);
```

Comment:
@cahrens this is ok. Although the design docs spec has 14px. We could possibly use font size variable here like we have in UXPL.